test/test_other.py

```diff
@@ -1687,18 +1687,38 @@ def test_module_stdin(self):
     self.emcc_args += ['--pre-js', 'pre.js']
     self.do_runf('module/test_stdin.c', 'hello, world!')
 
-  @also_with_noderawfs
+  # @also_with_noderawfs # NODERAWFS doesn't use library_tty.js for stdio streams
   @crossplatform
   def test_module_stdout_stderr(self):
     self.set_setting('FORCE_FILESYSTEM')
     create_file('pre.js', '''
+let stdout = [];
+let stderr = [];
+
+Module['stdout'] = (char) => stdout.push(char);
+Module['stderr'] = (char) => stderr.push(char);
+Module['postRun'] = () => {
+    assert(stderr.length === 0, 'stderr should be empty. \\n' +
+        'stderr: \\n' + stderr);
+    assert(UTF8ArrayToString(stdout, 0).startsWith('hello, world!'), 'stdout should start with the famous greeting. \\n' +
+        'stdout: \\n' + stdout);
+}
+''')
+    self.emcc_args += ['--pre-js', 'pre.js']
+    self.do_runf('hello_world.c')
+
+  # @also_with_noderawfs # NODERAWFS doesn't use library_tty.js for stdio streams
```

Comment:
Just remove this line?

Reply:
Removed with commit faac3bc62820b63205890b91ac513b0c035d71b1.